### PR TITLE
Add ControlCollection tests

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Add `ControlCollection` tests

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2273)

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.GetParent.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+using System;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [DllImport(Libraries.User32, ExactSpelling = true)]
+        public static extern IntPtr GetParent(IntPtr hWnd);
```
Comment (RussKie) [inline src/Common/src/Interop/User32/Interop.GetParent.cs]: Should we deleting definitions from some other file?

Comment (RussKie) [reply src/Common/src/Interop/User32/Interop.GetParent.cs]: Actually, why are we adding this definitions here at all?

Comment (hughbe) [reply src/Common/src/Interop/User32/Interop.GetParent.cs]: It’s extracted from #2272. Intention is to make sure that we call SetParent when we have a handle

Comment (RussKie) [reply src/Common/src/Interop/User32/Interop.GetParent.cs]: So should #2272 be merged first?

**Pull request merged**